robyn/scaffold/no-db/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11-bookworm
```

Comment:
python dockerfile because that way I didn't have to install pip
3.11 was the latest stable version
bookworm is the latest stable debian. I tried Alpine and it didn't work